### ISSUE TITLE
New JSON-file manifests don't include HistoricalStats

### ISSUE THREAD

**Issue by joshhunt** [Bungie-net/api#831]
The new JSON-file manifests don't include HistoricalStats. I presume this is related to them not actually having a hash (and instead have statId)? Would be nice if they were in there as well.

**Comment by vthornheart-bng** [comment]
Indeed, I'd like to start returning those too.  There's some work that needs to be done to make that happen - this may end up being a wishlist item for the next "major" revision of the API, as what I'd really like to see happen is standardizing how stats and their definitions are returned to work like the rest of the Destiny API (hash identifiers, fixed static definitions, etc...)  But that may end up being a pipe dream with our current resources and available time.
